scripts/popup.js

```diff
@@ -0,0 +1,140 @@
+  const wrapper = document.getElementById('wrapper');
+  let html = `
+    <div class="workDetailContainer ">
+    <!-- body -->
+    <div class="workDetailBody">
+    <div class="workDetailBodyHeader">
+        <h1 class="worksBodytitle">${projects[id - 1].title}</h1>
+        <i class="" id="workDetailClose"><img src="./images/work-detail-cancel.png" alt="icon-cancel"></i>
+        </div>
+        <!-- description -->
+        <div class="frame2">`;
+  projects[id - 1].tags.forEach((tag, i, arr) => {
+    html += `<span class="name">${tag}</span>`;
+    if (arr.length - 1 !== i) { html += '<span class="dot"></span>'; }
+  });
+  html += ` </div>
+        <img src="${projects[id - 1].img}" alt="${projects[id - 1].title}" class="workDetailImg">
+        <div class="workDetailInstructions">
+        <p class="workDescription workDetailDescription">
+           ${projects[id - 1].detail}
+        </p>
+        <!-- categories -->
+        <div>
+        <ul class="worksCategories">`;
+  projects[id - 1].categories.forEach((category) => {
+    html += `<li>${category}</li>`;
+  });
+  html += `</ul>
+        <hr>
+        <div class="workDetailFooter">
+        <button id="actionbtn" class="worksBtn worksDetailBtn worksDetailBtn " href="${projects[id - 1].liveVersion}">See live
+        <i class="" id=""><img src="./images/workDetailIcon1.png" alt="workDetailIcon1"></i>
+      </button>
+      <button id="actionbtn" class="worksBtn worksDetailBtn worksDetailBtn" href="${projects[id - 1].source}">See source
+      <i class="" id=""><img src="./images/workDetailIcon2.png" alt="workDetailIcon2" class="workDetailIcon2"></i>
+      </button>
+      </div>
+      </div>
+      </div>
+   </div>
+    <!-- end body -->
+ </div>`;
+  wrapper.style.display = 'block';
+  popup.innerHTML = html;
+  popup.style.display = 'block';
+  const workDetailClose = document.getElementById('workDetailClose');
+  workDetailClose.addEventListener('click', () => {
+    wrapper.style.display = 'none';
+    popup.style.display = 'none';
+  });
+}
+projects.forEach((project) => {
+  let html = `<div class="worksContainer " id="work${project.id}">
+   
+   <img src="${project.img}" alt="${project.title}" class="worksImg">
+   <!-- body -->
+   <div class="worksBody">
+       <h1 class="worksBodytitle">${project.title}</h1>
+       <!-- description -->
+       <div class="frame2">`;
+  project.tags.forEach((tag, i, arr) => {
+    html += `<span class="name">${tag}</span>`;
+    if (arr.length - 1 !== i) { html += '<span class="dot"></span>'; }
+  });
+  html += ` </div>
+       <p class="workDescription">
+          ${project.description}
+       </p>
+       <!-- categories -->
+       <ul class="worksCategories">`;
+  project.categories.forEach((category) => {
+    html += `<li>${category}</li>`;
+  });
+  html += `</ul>
+       <button class="worksBtn" id="${project.id}">See Project</button>
+  </div>
+   <!-- end body -->
+</div>`;
```

Comment:
- [ ] For some reason, the popup still does not match the project requirements as requested by the previous reviewer. the buttons still occur outside the popup "body" Kindly look into why this is the case.

**This is how I am seeing them:**

![image](https://user-images.githubusercontent.com/86318284/218138505-f039581b-ea90-44cc-bac6-a8ff8daf2867.png)